readthedocs/builds/tasks.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+import logging
+import shutil
+
+from django.db.models import Q
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from readthedocs.builds.constants import BUILD_STATE_FINISHED
+from readthedocs.builds.models import Version, Build
+from readthedocs.doc_builder.constants import DOCKER_LIMITS
+from readthedocs.projects.constants import LOG_TEMPLATE
+
+from readthedocs.worker import app
+
+
+log = logging.getLogger(__name__)
+
+
+@app.task(queue='web')
+def fileify(version_pk, commit):
+    """
+    Create ImportedFile objects for all of a version's files.
+
+    This is so we have an idea of what files we have in the database.
+    """
+    version = Version.objects.get(pk=version_pk)
+    project = version.project
+
+    if not commit:
+        log.info(
+            LOG_TEMPLATE.format(
+                project=project.slug,
+                version=version.slug,
+                msg=(
+                    'Imported File not being built because no commit '
+                    'information'
+                ),
+            )
+        )
+        return
+
+    path = project.rtd_build_path(version.slug)
+    if path:
+        log.info(
+            LOG_TEMPLATE.format(
+                project=version.project.slug,
+                version=version.slug,
+                msg='Creating ImportedFiles',
+            )
+        )
+        from readthedocs.projects.tasks import _manage_imported_files
```

Comment:
Looks like this isn't used on the `projects.tasks` module, so it can be moved too